geyser-plugin-manager/src/geyser_plugin_manager.rs

```diff
@@ -386,6 +386,7 @@ mod tests {
     }
 
     #[test]
+    #[ignore]
```

Comment:
Is it OK that geyser plugins could be broken on master?

Reply:
nope.. ;)

I'll try to repro this as well later.